redash/utils/parameterized_query.py

```diff
@@ -33,17 +33,59 @@ def _parameter_names(parameter_values):
     return names
 
 
+def _is_date(string):
+    try:
+        parse(string)
+        return True
+    except ValueError:
+        return False
+
+
+def _is_date_range(obj):
+    try:
+        return _is_date(obj["start"]) and _is_date(obj["end"])
+    except (KeyError, TypeError):
+        return False
+
+
 class ParameterizedQuery(object):
-    def __init__(self, template):
+    def __init__(self, template, schema={}):
```

Comment:
```suggestion
    def __init__(self, template, schema=None):
        self.schema = schema or {}
```
Same issue with mutable types as above.